process/smartContract/hooks/blockChainHook.go

```diff
@@ -97,21 +111,24 @@ func NewBlockChainHookImpl(
 	}
 
 	blockChainHookImpl := &BlockChainHookImpl{
-		accounts:              args.Accounts,
-		pubkeyConv:            args.PubkeyConv,
-		storageService:        args.StorageService,
-		blockChain:            args.BlockChain,
-		shardCoordinator:      args.ShardCoordinator,
-		marshalizer:           args.Marshalizer,
-		uint64Converter:       args.Uint64Converter,
-		builtInFunctions:      args.BuiltInFunctions,
-		compiledScPool:        args.CompiledSCPool,
-		configSCStorage:       args.ConfigSCStorage,
-		workingDir:            args.WorkingDir,
-		nilCompiledSCStore:    args.NilCompiledSCStore,
-		nftStorageHandler:     args.NFTStorageHandler,
-		globalSettingsHandler: args.GlobalSettingsHandler,
-		enableEpochsHandler:   args.EnableEpochsHandler,
+		accounts:                        args.Accounts,
+		pubkeyConv:                      args.PubkeyConv,
+		storageService:                  args.StorageService,
+		blockChain:                      args.BlockChain,
+		shardCoordinator:                args.ShardCoordinator,
+		marshalizer:                     args.Marshalizer,
+		uint64Converter:                 args.Uint64Converter,
+		builtInFunctions:                args.BuiltInFunctions,
+		compiledScPool:                  args.CompiledSCPool,
+		configSCStorage:                 args.ConfigSCStorage,
+		workingDir:                      args.WorkingDir,
+		nilCompiledSCStore:              args.NilCompiledSCStore,
+		nftStorageHandler:               args.NFTStorageHandler,
+		globalSettingsHandler:           args.GlobalSettingsHandler,
+		enableEpochsHandler:             args.EnableEpochsHandler,
+		gasSchedule:                     args.GasSchedule,
+		crtNumberOfBuiltInFunctionCalls: 0,
```

Comment:
since the default values for uint64s are 0, we can remove L130, L131
Otherwise, you should also add the crtNumberOfTransfers in this list

Reply:
done